src/packages/core/components/input-tree/input-tree.element.ts

```diff
@@ -0,0 +1,130 @@
+import { css, html, customElement, property } from '@umbraco-cms/backoffice/external/lit';
```

Comment:
I think we should put this component into the tree module as we will do with the `content type input`. You will find a tree folder inside the core package.